packages/test/test-end-to-end-tests/src/test/codePropsal.spec.ts

```diff
@@ -194,7 +193,7 @@ describeNoCompat("CodeProposal.EndToEnd", (getTestObjectProvider) => {
 		const maps: ISharedMap[] = [];
 		for (const container of containers) {
 			if (!container.closed) {
-				const dataObject = await requestFluidObject<ITestFluidObject>(container, "default");
+				const dataObject = (await container.getEntryPoint()) as ITestFluidObject;
```

Comment:
we should avoid casting, which should be possible here
```suggestion
				const maybeDataObject: FluidObject<ITestFluidObject> = await container.getEntryPoint();
				dataObject = maybeDataObject.ITestFluidObject
```

Reply:
Oh, that's nice